tests/rules/import-alias.test.ts

```diff
@@ -79,19 +78,13 @@ function runTests(platform: "win32" | "posix") {
                 code: `export * from '../src-app/rules/potato';`,
                 filename: "src/test.ts",
             },
-            // selects correct alias despite #src being a partial match
-            // and comes first/is shorter as an alias
-            {
-                code: `export * from '../src-test/rules/potato';`,
-                filename: "src/test.ts",
-            },
```

Comment:
Did the test cause issues with the new implementation? If not, let's put them back and we'll fix it in a separate commit/PR.

Think I saw another case where this happened somewhere lower down as well

Reply:
i think i remove it because the comment fon't make sense, and case where we validate that not alias match are already done in this rule tests